main/src/ca/uwaterloo/flix/api/Flix.scala

```diff
@@ -80,6 +80,7 @@ class Flix {
     * The set of known Java classes and interfaces.
     */
   private var knownClassesAndInterfaces: MultiMap[List[String], String] = getJavaPlatformClassesAndInterfaces()
+//  private var knownClassesAndInterfaces: AvailableClasses = AvailableClasses() TODO: which root should we use?
```

Comment:
We should construct and maintain it here. See how knownClassesAndInterfaces is used.